pages/index.js

```diff
@@ -286,7 +286,7 @@ class Index extends React.Component {
                 body: JSON.stringify({
                   query:emitOnlineEvent,
                   variables: {
-                      username:this.state.username,
+                      ...username:this.state.username,
```

Comment:
Use a more structured approach like an object for spreading properties instead of directly spreading 'this.state.username' for better readability and maintainability.